tendrl/commons/message.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from inspect import getframeinfo
+from inspect import stack
+import uuid
+
+
+class Message(object):
+    def __init__(self, message):
+        try:
+            if "message_id" not in message:
+                self.message_id = str(uuid.uuid4())
+            else:
+                self.message_id = message["message_id"]
+            self.timestamp = message["timestamp"]
```

Comment:
Shouldn't be timestamp be automatically generated when the object is instantiated? Otherwise, we will have to import datetime in every module